### DISCUSSION TITLE
SQL scripts for creating schema

### DISCUSSION THREAD

**Discussion by vpasechnik** [ankicommunity/djankiserv#25] [Q&A] [answered]
Hi,
I'm trying to setup project using mysql, but when I try to login via admin console I'm getting:
```
The above exception ((1146, "Table 'djankiserv.auth_user' doesn't exist")) was the direct cause of the following exception:
```
and as I see my djankiserv database is empty. I see only sqlite scripts in git, and they do not have auth_user table. Where I can get proper sql scripts for djankiserv?

**Comment by AntonOfTheWoods** [comment, marked as answer]
Hi @vpasechnik, the `sqllite3` scripts are for creating the export files (which anki needs to download at various points). The `mysql` (or `pgsql`) tables are created via code. Have a look at https://github.com/ankicommunity/djankiserv/blob/master/src/djankiserv/unki/database.py for more detail.

`djankiserv` does, however, use the normal `django` user system, meaning the table that is being complained about. These should get created via the normal `manage.py migrate` database workflow, which is the same for any backend. I just tested with `mysql` and everything should get created as normal.
